packages/adapter/lib/adapter/adapter.js

```diff
@@ -2591,7 +2591,7 @@ function Adapter(options) {
 
             params = params || {};
 
-            if (design === 'system' && !params.startkey.length && !params.endkey.length) {
```

Comment:
Params endkey \u9999 should be handled as falsy too.

Reply:
really? Then the caller set something and so I would assume he knows what he's doing, or?!

Reply:
But can he get anything else expect what we set here?

Reply:
Ok, but then I do not need to check, we just hard overwritewhatever came in

Reply:
Done. We now hard overwrite

Reply:
Why hard override everytime. For me the reason was, if someone does startkey: '', endkey: '\u9999' it will be a * search, so give me everything. And if someone wants everything on e. g. enums then we can replace it. But if he wanted startkey: 'enum.rooms, endkey: 'enum.rooms.\u9999 then we shouldn't return everything, he does not expect enum.functions.lights with this call.